tests/test_oauth2_reddit.py

```diff
@@ -108,6 +110,24 @@ def test_scope_modlog(self):
         self.assertEqual(num, len(list(result)))
 
     @betamax()
+    def test_scope_modothers_modself(self):
+        subreddit = self.r.get_subreddit(self.sr)
+        self.r.refresh_access_information(self.refresh_token['modothers'])
+        subreddit.add_moderator(self.other_user_name)
+
+        # log in as other user
```

Comment:
I'm not sure why all of that is necessary, aside from calling `refresh_access_information` again with the `modself` token. The same client token and secret should be usable by multiple accounts via their tokens. Perhaps I am mistaken.